R/doeResponseSurfaceMethodology.R

```diff
@@ -365,8 +361,7 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
         plot$dependOn(c("rsmResponseVariables",
                         "rsmBlocks",
                         "contour", "psi", "theta","pairs", "Component", "Point"))
-        contourPlot[[paste0("plotObject", col, sep = "")]] <- plot
-
+        jaspResults[[paste("contourPlot", lol, sep = "")]][[paste0("plotObject", col, sep = "")]] <- plot
```

Comment:
```suggestion
        jaspResults[[paste0("contourPlot", lol)]][[paste0("plotObject", col)]] <- plot
```
`paste0` does not take a sep argument, but is identical to `paste(..., sep = "")`.

Can you fix all occurrences of this?